commons/src/main/res/values/strings.xml

```diff
@@ -214,6 +214,9 @@
     <string name="add_image">Add Image</string>
     <string name="add_signature">Add signature</string>
     <string name="add_file">Add file</string>
+    <string name="download">Download</string>
+    <string name="scan">Scan</string>
+    <string name="qr_code">QR code</string>
```

Comment:
As this texts are related to the form why not add the to the string in the form module?

Reply:
good idea, I've just updated with the changes